app/src/main/java/org/helfoome/util/SnackBarTopDown.kt

```diff
@@ -0,0 +1,29 @@
+package org.helfoome.util
+
+import android.content.Context
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
+import androidx.viewbinding.ViewBinding
+import org.helfoome.R
+import org.helfoome.presentation.custom.SnackBarView
+
+object SnackBarTopDown {
+    fun makeSnackBarTopDown(context: Context, binding: ViewBinding, id: Int, snackBarText: String) {
+
+        val animation = AnimationUtils.loadAnimation(context, R.anim.anim_snackbar_top_down)
+
+        val snackBarView = binding.root.findViewById<SnackBarView>(id)
+        snackBarView.animation = animation
+        snackBarView.setText(snackBarText)
```

Comment:
코틀린은 프로퍼티 기능이 잘 되어있으니까, 요렇게 표현하면 좀 더 간결할 것 같아요!

```suggestion
        snackBarView.text = snackBarText
```

Reply:
너무좋네요